Demo/Sources/Players/PlaybackView.swift

```diff
@@ -287,7 +327,10 @@ private struct TimeBar: View {
             routePickerView()
             TimeSlider(player: player, progressTracker: progressTracker)
             LiveLabel(player: player, progressTracker: progressTracker)
+            FullScreenButton(layout: $layout)
         }
+        // Prevent taps from going through in transparent areas
+        .background(Color(white: 1, opacity: 0.0001))
```

Comment:
```suggestion
        .contentShape(Rectangle())
```

Reply:
Sadly the behavior is not the same. Taps can still go through with `contentShape(_:)`, which means tapping on a transparent area next to the full screen double arrow will trigger not only the full screen toggle button, but also the user interface visibility gesture, which is awkward.